src/ui/views/ConfiguratorView/index.tsx

```diff
@@ -423,6 +429,26 @@ const ConfiguratorView: FunctionComponent<ConfiguratorViewProps> = (props) => {
     return deviceType === DeviceType.ExpressLRS && isTX;
   }, [deviceType, isTX]);
 
+  const eraseSupported = useMemo(() => {
+    if (!deviceTarget || !device) {
+      return false;
+    }
+    if (deviceTarget.flashingMethod === FlashingMethod.UART) {
+      return true;
+    }
+    if (
+      deviceTarget.flashingMethod === FlashingMethod.BetaflightPassthrough &&
+      device.platform === 'esp8285'
```

Comment:
ESP32 based receivers are not supported in this case?

Reply:
No, because we are doing the flashing from our firmware rather than the boot rom.